### ISSUE TITLE
Question: Is there a parameter to change the data augmentation probability?

### ISSUE THREAD

**Issue by wellescastro** [jpuigcerver/PyLaia#14]
Hi Joan Puigcerver

I'm trying to figure out how I can modify the probability in which the distortions are applied to reproduce the exact same setup used in your work on ICDAR 2017. I am checking the code of the Dilation and Erosion morphologies techniques but I did not understand yet how I can modify the chance of applying these transformations to 0.5 (actually I don't know if this is already being made), maybe I can use the alpha and beta parameters.

Thanks,
Dayvid

**Comment by jpuigcerver** [comment]
Hi Dayvid,

Notice that the distorter in PyLaia is not the same as Laia's. The reason is two-fold:
- Having simpler implementation.
- Having less hyperparameters to tune.

In PyLaia, the distorter is part of the image processing applied during training. See:
https://github.com/jpuigcerver/PyLaia/blob/fe44ee6f663270cb01f8d8bd4da10867741f7cf7/pylaia-htr-train-ctc#L140

Currently, the only distortion that can be used is RandomBetaAffine with its default parameters (see https://github.com/jpuigcerver/PyLaia/blob/fe44ee6f663270cb01f8d8bd4da10867741f7cf7/laia/data/transforms/vision/random_beta_affine.py#L12).

If you want to change the distortion parameters or add more distortions (e.g. morphology), the easiest option (at this moment) is to modify the pylaia-htr-train-ctc script. 

In any case, you won't be able to exactly match the results from the ICDAR 2017 paper. However, I've used the current version of PyLaia for different experiments on IAM and I have obtained better results than the ones published in the paper (both with and without external language models).
